src/lvl_script.c

```diff
@@ -6921,6 +6936,7 @@ const struct CommandDesc command_desc[] = {
   {"USE_SPELL_ON_CREATURE",             "PCAAN   ", Cmd_USE_SPELL_ON_CREATURE, NULL, NULL},
   {"SET_HEART_HEALTH",                  "PN      ", Cmd_SET_HEART_HEALTH, NULL, NULL},
   {"ADD_HEART_HEALTH",                  "PNN     ", Cmd_ADD_HEART_HEALTH, NULL, NULL},
+  {"CREATURE_ENTRANCE_LEVEL",           "N       ", Cmd_CREATURE_ENTRANCE_LEVEL, NULL, NULL},
```

Comment:
Looking at this again, I wonder why you don't have a P(layer) parameter. Wouldn't it make much more sense if you could do it on a per-player basis, or ALL_PLAYERS if need be?

Reply:
I'm not sure it's possible to have this as a player setting. Where would we store the value?

Reply:
Well, SET_GENERATE_SPEED, which also determines how the portals work, are stored (per player actually, after calculating the score) in the DUNGEON structure. So, I would say normally the Dungeon structure, and if we can't expand that (which we can't) in DungeonAdd.

Reply:
It now has a player setting.